src/selfplay/game.cc

```diff
@@ -63,6 +67,8 @@ void SelfPlayGame::Play(int white_threads, int black_threads, bool training,
                         bool enable_resign) {
   bool blacks_move = false;
 
+  int ply_count = 0;
```

Comment:
This is available through `tree_[0]->GetPositionHistory().GetLength()`, probably better than adding a counter.